src/FsAutoComplete.Core/UnionPatternMatchCaseGenerator.fs

```diff
@@ -211,6 +211,19 @@ let private tryFindPatternMatchExprInParsedInput (pos: Position) (parsedInput: P
             | _ -> None
           else
             None)
+        |> Option.orElseWith (fun () ->
+          if synMatchClauseList.IsEmpty then
+            match debugPoint with
```

Comment:
Any reason you went with this extra check for the debug point.
I don't think it matters, checking if `synMatchClauseList.IsEmpty` probably is enough.

Reply:
I think I copied this check from preceding "branch" of orElseWith call chain. Honestly, I don't even know what DebugPointAtBinding is here for :sweat_smile: